### PR TITLE
Completed javascript-functions exercise. - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Screenshot 2023-05-09 at 4 17 39 PM

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  return width * height;
+}
+console.log('Value of function getArea w/ argument of 4, 36:', getArea(4, 36));
+
+// getFirstName(person)
+function getFirstName(person) {
+  person = {
+    firstName: 'Jesus',
+    lastName: 'Araujo',
+  };
```
Comment (shawnkost) [inline javascript-functions/main.js:27]: You don't want to define `person` inside of this function, you want to pass this object when you call the function

File: javascript-functions/main.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  return person.firstName;
+}
+console.log(
+  'Value of getFirstName w/ argument of Jesus, Araujo:',
+  getFirstName()
+);
+
+// getLastElement(array)
+function getLastElement(array) {
+  array = ['mango', 'pineapple', 'apples'];
```
Comment (shawnkost) [inline javascript-functions/main.js:37]: You don't want to define `array` inside of this function, you want to pass this array when you call the function

**Review by shawnkost — CHANGES_REQUESTED** [review]
A couple changes here Jesus!

File: javascript-functions/main.js (hunk 1 above)
Comment (j-alejandro-araujo) [reply javascript-functions/main.js]: Screenshot 2023-05-09 at 7 27 56 PM

Fixed, good catch!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**